libraries/botbuilder-core/src/cardFactory.ts

```diff
@@ -257,15 +286,13 @@ export class CardFactory {
 
     /**
      * Returns an attachment for a thumbnail card.
-     *
      * @param title The card title.
-     * @param text Optional. The card text.
      * @param images Optional. The array of images to include on the card. Each element can be a
      *      [CardImage](ref:botframework-schema.CardImage) or the URL of the image to include.
      * @param buttons Optional. The array of buttons to include on the card. Each `string` in the array
      *      is converted to an `imBack` button with a title and value set to the value of the string.
      * @param other Optional. Any additional properties to include on the card.
-     * 
+     * @returns An [Attachment](xref:botframework-schema.Attachment).
```

Comment:
Shouldn't we add a return doc to the previous methods too?

Reply:
I only added it to the ones we modified, i will add it to the rest